ape_infura/__init__.py

```diff
@@ -12,11 +12,22 @@ def providers():
 
 
 def __getattr__(name: str):
-    if name == "NETWORKS":
+    if name == "Infura":
+        from ape_infura.provider import Infura
+
+        return Infura
+    
+    elif name == "NETWORKS":
         from ape_infura.utils import NETWORKS
 
         return NETWORKS
 
     import ape_infura.provider as module
```

Comment:
should just raise attr err here i guess? since we are enumerating them all above?

Reply:
yeee makes sense, fixed